typewritten.zsh

```diff
@@ -102,7 +106,7 @@ tw_redraw() {
       tw_git_arrow_info=" $tw_arrow %F{$tw_git_branch_color}$tw_git_info"
     fi;
     if [ "$tw_layout" = "pure" ]; then
-      PROMPT="$BREAK_LINE$tw_home_relative_wd$tw_git_arrow_info$BREAK_LINE$tw_env_prompt"
+      PROMPT="$BREAK_LINE$tw_git_relative_wd$tw_git_arrow_info$BREAK_LINE$tw_env_prompt"
```

Comment:
**suggestion(non-blocking)**: Since we now only use the `tw_git_relative_wd` variable for all prompts, we could rename it to something a little more generic. The function called above essentially returns what directory is displayed. We could name it `tw_get_displayed_wd` and rename this variable to be `tw_displayed_wd`.